code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -496,6 +511,21 @@
 			return 0
 	return 1
 
+/mob/living/simple_animal/proc/attack_threshold_check(damage, damagetype)

```

Comment:
I dunno the PR it was changed in, but: https://github.com/tgstation/-tg-station/blob/61d23d8c1112fdc45bce2e8422122ca136aa248b/code/modules/mob/living/simple_animal/simple_animal.dm#L356